src/utils/securityCheck.js

```diff
@@ -0,0 +1,22 @@
+export async function checkAuthorityRequests(idOfUserForEdit, requesterId){
+// TODO - Export most of check from userService
+// function for approving from Valeri Getsov
```

Comment:
The idea is good, but it's not dynamic. 
You can pass a third argument to check whatever you want (like, if you want only admin rights or admin and organizer rights or only organizer rights, whatever you want and need -> you can pass a third param "role" or "roles" which can be optional, for example, where you can pass the needed role auth if needed this auth)